tensorflow_serving/g3doc/serving_basic.md

```diff
@@ -172,7 +172,7 @@ saved_model.pb variables
 
 Each version sub-directory contains the following files:
 
-  * `saved_model.pb` is the serialized tensorflow::SavedModel. It includes the
+  * `saved_model.pb` is the serialized tensorflow::SavedModel. It includes
   the one or more graph definitions of the model, as well as metadata of the
```

Comment:
Thanks! Let's also remove this second "the" at the beginning of line 176. "It includes one or more graph definitions of the model, as well as metadata..."